app/models/stripe-account.js

```diff
@@ -0,0 +1,16 @@
+import Model from 'ember-data/model';
+import attr from 'ember-data/attr';
+import { belongsTo } from 'ember-data/relationships';
+
+export default Model.extend({
+  businessName: attr(),
```

Comment:
I don't think it matters.  `DS.StringTransform` will convert integers to strings.  The `boolean` transform will convert strings (other than `"true"` or `"t"`) to false.  But you really shouldn't need the transforms, since we control our API and how our information is serialized.